### ISSUE TITLE
Allow to use `markdownlint` format of `keep-a-changelog` version `2.2.0`

### ISSUE THREAD

**Issue by Ndpnt** [superfaceai/release-changelog-action#269]
Hi,

Thanks for this useful GitHub Action to generate changelog.

We need to generate a changelog that respect [markdownlint](https://github.com/DavidAnson/markdownlint) rules.
An update has been made on `keep-a-changelog` module in version `2.2.0` in this way. 
Could you allow to specify the output format as parameter of this GitHub action?

Something like this:
```
name: Get version changelog
  on:
    ...

  jobs:
    get-changelog:
      steps:
        - uses: actions/checkout@v2
        - name: Get changelog
          id: get-changelog
          uses: @superfaceai/release-changelog-action@v2
          with:
            path-to-changelog: CHANGELOG.md
            version: 1.0.0
            format: markdownlint
        - name: Log changelog
          run: echo ${{ steps.get-changelog.outputs.changelog }}
```

**Comment by MattiSG** [comment]
That would be great indeed!

@janhalama FYI, following https://github.com/oscarotero/keep-a-changelog/issues/28, the format is [documented here](https://github.com/oscarotero/keep-a-changelog#custom-output-format) 🙂

**Comment by janhalama** [comment]
I like your proposal.  I will add the `format` parameter as you suggest with `markdownlint` and `compact` (default) options.

**Comment by janhalama** [comment]
Implemented in #270.

**Comment by Ndpnt** [comment]
Thanks 🙏

**Comment by janhalama** [comment]
@Ndpnt, @MattiSG the markdownlint  format support has been released in v2.1.1 the v2 tag has been updated to the latest version

**Comment by MattiSG** [comment]
Amazing, thanks!
